src/com/codefortomorrow/intermediate/chapter11/solutions/IsPrime.java

```diff
@@ -0,0 +1,42 @@
+package com.codefortomorrow.intermediate.chapter11.solutions;
+
+/*
+Write a method called isPrime which returns
+true if the given integer is prime and false otherwise.
+Bonus points for writing JavaDoc comments for the method.
+
+Test your isPrime method in the main method by
+calling it at least 3 times and printing the result.
+
+Solution from Listing 6.7, Introduction to Java Programming (Comprehensive),
+10th ed. by Y. Daniel Liang
+*/
+
+public class IsPrime {
+
+    public static void main(String[] args) {
+        System.out.println(isPrime(5)); // true
+        System.out.println(isPrime(1)); // false
+        System.out.println(isPrime(56)); // false
+    }
+
+    /**
+     * Check whether number is prime
+     * @param number  number to check if prime
+     * @return true if the number is prime
+     */
+    public static boolean isPrime(int number) {
+        if (number <= 1) {
+            return false;
+        }
+
+        // only need to check divisors up to number / 2
```

Comment:
technically you only need to check up to the square root of the number

Reply:
That's a good point!